src/DonationForms/Properties/FormSettings.php

```diff
@@ -311,6 +323,10 @@ public static function fromArray(array $array): self
             $array['designSettingsTextFieldStyle']
         ) : DesignSettingsTextFieldStyle::DEFAULT();
 
+        $self->designSettingsImageColor = $array['designSettingsImageColor'];
+
+        $self->designSettingsImageOpacity = $array['designSettingsImageOpacity'];
```

Comment:
We'll probably want to do a nullish conditional on these to fallback to when there is no image color or opacity

Reply:
Resolved: b37aa50cafcf0ebf2552a8a66d8dabce295a7c08